### DISCUSSION TITLE
Some mistakes in UI Walkthrough

### DISCUSSION THREAD

**Discussion by NestorMenta** [uxbox/uxbox#608] [Q&A] [answered]
Hi!

I'm giving a look to the UI Walkthrough file to know Penpot better, and I've notice some mistakes:

- In the **Workspace page** and in the **Asset Artboar**d: Colors, and Tipographies have the same description as Graphics.
- In the **Interactions artboard**, you say "Once selected the prototype mode at the left sidebar...". It was confusing to me, because the Prototype mode is actually in the right, next to Design.
- In the **Prototype mode Artboard** I found a small typo "Preview static **desings**"

And that's all I've been able to spot. I'm going yo give it a try because It seems an awesome app and I will gave you any feedback I could. Thanks for this huge work!

**Comment by myfunnyandy** [comment, marked as answer]
With "at the left sidebar..." we meant the other left (apologies for the bad joke). Thanks for reporting those typos, we'll fix them soon ([Taiga issue](https://tree.taiga.io/project/penpot/issue/1162)).

Glad you are giving Penpot a try, tell us if you find anything else :)
